### ISSUE TITLE
SNAPSHOT dependency in 1.5.5.RELEASE

### ISSUE THREAD

**Issue by david-kerwick** [spring-projects/spring-boot#9871]
Looks like the 1.5.5 release has a snapshot dependency in it.

```
<dependency>
        <groupId>org.springframework.data</groupId>
        <artifactId>spring-data-mongodb-log4j</artifactId>
        <version>1.10.5.BUILD-SNAPSHOT</version>
</dependency>
```

Causes release plugins to fail.

Thanks
David

**Comment by wilkinsona** [comment]
The problem is in Spring Data's bom. Unfortunately, our enforcer rules didn't catch this one as we don't depend on `org.springframework.data:spring-data-mongodb-log4j`.

@olivergierke Can was please have an Ingalls-SR-2 bom?

**Comment by wilkinsona** [comment]
We plan to release 1.5.6 tomorrow to fix this. In the interim, anyone who is affected can work around the problem by adding the following to their application's pom:

```xml
<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>org.springframework.data</groupId>
            <artifactId>spring-data-mongodb-log4j</artifactId>
            <version>1.10.5.RELEASE</version>
        </dependency>
    </dependencies>
</dependencyManagement>
```

**Comment by masu1402** [comment]
For me the issue is still relevant, but with version 1.5.6.RELEASE. What happens is that if I specify to use version 1.5.6.RELEASE my Maven Dependencies library contains only a single jar. If I change to 1.5.5 or .1.4.7 all the .jar files that are supposed to be downloaded by the manager is fetched as intended (working). Is there still issues with this in 1.5.6 already recorded or should I start a new issue? @david-kerwick @rajadileepkolli @wilkinsona

**Comment by snicoll** [comment]
I didn't get the issue you are describing nor the link with this issue. This particular issue is fixed. Perhaps we can chat on gitter?

**Comment by david-kerwick** [comment]
Hi @masu1402  the fix worked and I've had no issues with 1.5.6.

**Comment by masu1402** [comment]
Removed my comment. Had nothing to do with this issue.
